src/app/subscriptions/routes.ts

```diff
@@ -0,0 +1,23 @@
+import type { RouteRecordRaw } from 'vue-router'
+export const routes = (): RouteRecordRaw[] => {
+  return [
+    {
+      path: 'subscription/:subscription',
+      name: 'subscription-summary-view',
+      redirect: { name: 'subscription-summary-overview-view' },
+      component: () => import('@/app/subscriptions/views/SubscriptionSummaryView.vue'),
+      children: [
+        {
+          path: 'overview',
+          name: 'subscription-summary-overview-view',
+          component: () => import('@/app/subscriptions/views/SubscriptionSummaryOverviewView.vue'),
+        },
+        {
+          path: 'config',
```

Comment:
This needs to be double checked (@jakubdyszkiewicz will know) but AFAIR only the first subscription has a config as it's the one already shown in the `Config` tab in the overview. Therefore the tab should only show the overview

Reply:
Ahhh right gotcha, was super unsure about this (see questions in PR I crossed out 🫤 )

Looking at the issue this PR will hopefully close (https://github.com/kumahq/kuma-gui/issues/2164), it says this:

> We should also show: the version at the top. Finally each subscription should have a copy raw button to just get the json/yaml of the subscription. We should also show if the version is not the latest to suggest upgrade to customers.

So this means we want to show the subscription YAML itself not its JSON config I think, will check in with @jakubdyszkiewicz also, but I'm pretty sure this is what we need to do.

Reply:
I checked in with @jakubdyszkiewicz offline, generally it is the case that only one subscription has a `.config` field.

So I'm pretty sure I just misunderstood the original ask.

I've already changed this so we show the subscriptions YAML in the tab instead, so I'm gonna resolve this convo.

but shout if there's anything else to do here.